tests/app/template/test_rest.py

```diff
@@ -210,6 +210,34 @@ def test_should_raise_error_on_create_if_no_permission(
     assert json_resp['message'] == expected_error
 
 
```

Comment:
Can we have a test for creating a template when service has CHOOSE_POSTAGE permission and postage is set for template?

Reply:
And a similar test for updating a template